guide/14-deep-learning/use_mistral_llm_for_text_classification_and_entity_recognition.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<p>To classify text using the mistral model, use the <code>predict</code> method from the <code>TextClassifier</code> class. The input to the method will be a text string or a list of text string.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>